lib/xcpretty/formatters/formatter.rb

```diff
@@ -173,10 +174,13 @@ def format_will_not_be_code_signed(message)
     end
 
     def format_other(text)
-      ""
+      if @unmatched and text and text.empty? == false and text.gsub(/\s+/, '').lstrip.empty? == false #and text != "\n" 
```

Comment:
Remove the part that is commented out if it is not needed.
`text.empty? == false` => `!text.empty?`.
`text.gsub(/\s+/, '').lstrip.empty? == false` => `!text.gsub(/\s+/, '').lstrip.empty?`.
Finally, the `text.empty?` check is already caught by the `text.gsub(/\s+/, '').lstrip.empty?` check. I think only the latter is needed, unless you have good reason for this optimization.
Also, add some unit tests for this?

Reply:
Just added some  tests, but can't run them due to `bundle install --path vendor/bundle` failing (`mkmf.rb can't find header files for ruby`) using macOS system ruby

Reply:
You're probably missing development environment / headers. Are you using rbenv or just clean macOS system ruby? I believe installing rbenv and installing some ruby version from there should solve the issue. We all use ruby 2.5.3

Reply:
got some issue with bundle command using system ruby instead of env one. rake spec now runs fine